### ISSUE TITLE
Cannot Build Buildings

### ISSUE THREAD

**Issue by loganrk9** [Thalassicus/cep-bnw#297]
Using a custom civ and I cannot produce any buildings. Have been scouring forums and google for similar issues or a compatibility check to no avail. Are custom civs not able to use the mod?

**Comment by zimeron** [comment]
Been having the same issue.  Is there anything in your Lua.log that seems to be related? Mine has been pretty useless.

**Comment by loganrk9** [comment]
Hey sorry I was out of town so I was not really checking all this stuff. To your point though, no unfortunately that has not yielded any results. Is it too much to ask for custom civs with nice ai? :p

**Comment by zimeron** [comment]
Yeah, I started using the Community Patch instead.  It's a similar thing that is more tedious to install but is compatible with the JFD civs.

**Comment by loganrk9** [comment]
Awesome I will try that out thanks!
